dbms/src/Storages/Page/V3/LogFile/LogFilename.cpp

```diff
@@ -29,28 +29,29 @@ LogFilename LogFilename::parseFrom(const String & parent_path, const String & fi
     if (!startsWith(filename, LOG_FILE_PREFIX_TEMP) && !startsWith(filename, LOG_FILE_PREFIX_NORMAL))
     {
         LOG_INFO(log, "Ignore not log file [dir={}] [file={}]", parent_path, filename);
-        return {LogFileStage::Invalid, 0, 0, 0, ""};
+        return {LogFileStage::Invalid, 0, 0, 0, 0, ""};
```

Comment:
We'd better throw exception when meet unknown filename in the directory in `WALStoreReader::listAllFiles`